src/transactions/OperationFrame.cpp

```diff
@@ -47,7 +49,7 @@ getNeededThreshold(LedgerTxnEntry const& account, ThresholdLevel const level)
 
 shared_ptr<OperationFrame>
 OperationFrame::makeHelper(Operation const& op, OperationResult& res,
-                           TransactionFrame& tx)
+                           TransactionFrame& tx, size_t index)
```

Comment:
per my other comment elsewhere, `size_t` is overkill for this as we later have to downcast it. Better to just downcast it once (here). Not sure why it was `size_t` before in `TransactionFrame::makeOperation`.

Reply:
fixed